<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FSyncX</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="APPPage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="HomePage">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>82</y>
        <width>511</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="LOGO_2">
       <item row="0" column="0">
        <widget class="QLabel" name="LOGO_T">
         <property name="styleSheet">
          <string notr="true">font: 64pt &quot;Ink Free&quot;;
</string>
         </property>
         <property name="text">
          <string>FSyncX</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="LOGO_S">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
         </property>
         <property name="text">
          <string>Root folder synchronization tool based on TCP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnFileBase">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>330</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Fira Code&quot;;</string>
      </property>
      <property name="text">
       <string>I'm FileBase-Server &gt;&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFileSync">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Fira Code&quot;;</string>
      </property>
      <property name="text">
       <string>I'm FileSync-Client &gt;&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SyncPage">
     <widget class="QGroupBox" name="Quick">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>361</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Quick</string>
      </property>
      <widget class="QPushButton" name="BtnOpenSyncPath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>341</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Open Sync PATH</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Debug">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>381</width>
        <height>461</height>
       </rect>
      </property>
      <property name="title">
       <string>Dubug</string>
      </property>
      <widget class="QTextBrowser" name="TBrwSyncDebug">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>351</width>
         <height>431</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
      </widget>
     </widget>
     <widget class="QProgressBar" name="PBarCommon">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>530</y>
        <width>791</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
       </font>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="ConfigGBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>361</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Config</string>
      </property>
      <widget class="QProgressBar" name="PBarSyncConfig">
       <property name="geometry">
        <rect>
         <x>-30</x>
         <y>40</y>
         <width>411</width>
         <height>301</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Fira Code</family>
        </font>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="Sync">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>361</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Sync</string>
       </property>
       <widget class="QPushButton" name="BtnStartSync">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>341</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Sync</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="Connect">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>361</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Connect</string>
       </property>
       <widget class="QPushButton" name="BtnConnectToFBase">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect to FBase - Server</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LEditIP">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>241</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>192.168.1.5</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LEditPort">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>8888</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="SyncPath">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>361</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>SyncPath</string>
       </property>
       <widget class="QPushButton" name="BtnChoseSyncPath">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>32</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>ChosePath</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="TBrwSyncPath">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Code'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;NO PATH CHOOSE..&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="PageLOGO_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>751</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>FileSync- Client</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>ConfigGBox</zorder>
     <zorder>Quick</zorder>
     <zorder>Debug</zorder>
     <zorder>PBarCommon</zorder>
     <zorder>PageLOGO_2</zorder>
    </widget>
    <widget class="QWidget" name="BasePage">
     <widget class="QGroupBox" name="Quick_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>361</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Quick</string>
      </property>
      <widget class="QPushButton" name="BtnOpenBasePath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>341</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Open Base PATH</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="ConfigGBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>361</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Config</string>
      </property>
      <widget class="QProgressBar" name="PBarBaseConfig">
       <property name="geometry">
        <rect>
         <x>-30</x>
         <y>40</y>
         <width>411</width>
         <height>301</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Fira Code</family>
        </font>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>361</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>On Sever Listen</string>
       </property>
       <widget class="QPushButton" name="BtnStartListen">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>341</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Listen</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>361</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>GetHost</string>
       </property>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>50</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Share IP</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LEditServerPort">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>8888</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="TBrwServerHost">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>241</width>
          <height>21</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Code'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; color:#5f5f5f;&quot;&gt;Click [Refresh IP] to get...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnGetIP">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Refresh IP</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>361</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>BasePath</string>
       </property>
       <widget class="QPushButton" name="BtnChoseBasePath">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>32</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>ChosePath</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="TBrwBasePath">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Code'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#161616;&quot;&gt;NO PATH CHOOSE..&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QProgressBar" name="PBarCommon_4">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>530</y>
        <width>791</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
       </font>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="Debug_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>381</width>
        <height>461</height>
       </rect>
      </property>
      <property name="title">
       <string>Dubug</string>
      </property>
      <widget class="QTextBrowser" name="TBrwBaseDebug">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="PageLOGO">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>751</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>FileBase- Server</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionBackHome"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_use"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Eixt</string>
   </property>
  </action>
  <action name="actionBackHome">
   <property name="icon">
    <iconset resource="icons/APPicons.qrc">
     <normaloff>:/icon_backhome.png</normaloff>:/icon_backhome.png</iconset>
   </property>
   <property name="text">
    <string>BackHome</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="icons/APPicons.qrc">
     <normaloff>:/icon_Setting.png</normaloff>:/icon_Setting.png</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionHow_to_use">
   <property name="icon">
    <iconset resource="icons/APPicons.qrc">
     <normaloff>:/icon_HOW.png</normaloff>:/icon_HOW.png</iconset>
   </property>
   <property name="text">
    <string>How to use</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/APPicons.qrc"/>
 </resources>
 <connections/>
</ui>
